Resources/Prototypes/_Nuclear14/Entities/Structures/Wallmount/walldecor.yml

```
@@ -0,0 +1,179 @@
+# Flags
+- type: entity
+  parent: N14SignBase
+  id: N14FlagWallUS
+  name: flag
+  suffix: US
+  description: A US flag.
+  components:
+  - type: Sprite
+    sprite: _Nuclear14/Structures/Wallmounts/walldecor.rsi
+    state: flag_us
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)